.github/workflows/hf-llm-inference.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: 'Commit to build'
+        required: true
+        default: 'master'
```

Comment:
Branch names don't work quite right in Dockerfiles, so better to remove that default
```suggestion
```